src/bacnet/basic/object/nc.c

```diff
@@ -979,4 +972,72 @@ void Notification_Class_find_recipient(void)
         }
     }
 }
+
+void Notification_Class_Register_Destination (uint32_t device_id, uint16_t nc_instance, bool confirmed)
+{
+    bool empty_space = false;
+    bool already_registered = false;
+    BACNET_DESTINATION *RecipientEntry;
+    if (nc_instance >= NC_Info_Size)
+    {
+        fprintf(stdout, "NC instance is larger than maximum in the Simulator\n");
```

Comment:
Do we not use the stack for more than just the simulator? Would this logging be an issue?

Reply:
We will not use this function on the client side of the stack. Most of this file will not be built in the client side of the stack anyway as its within:

#if defined(INTRINSIC_REPORTING)

which is not set